CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## [3.0.4+portage-3.0.4] - 2022-02-25
+  - Update translation for some wording
+  
 ## [3.0.4+portage-3.0.3] - 2020-02-24
```

Comment:
We need to change this time traveling version as you did with the previous one ;)

Reply:
Sorry what is the version it supposed to be?

Reply:
Or I should just keep the 3.0.3 but change the date to today?